web/releases/3.10.json

```diff
@@ -1,132 +1,76 @@
-          "path": "Managing_Security_Compliance"
-        },
-        {
-          "title": "Administering Foreman",
-          "path": "Administering_Project"
-        },
-        {
-          "title": "Application centric deployment",
-          "path": "Deploying_Hosts_AppCentric"
-        },
-        {
-          "title": "Monitoring Foreman performance",
-          "path": "Monitoring_Project"
+    "katello": "4.12",
+    "state": "unsupported",
+    "builds": [
+        {
+            "title": "Katello on EL",
+            "header": "Foreman {FOREMAN_VER} and Katello {KATELLO_VER}",
+            "filename": "index-katello.html",
+            "sections": {
+                "Release notes and upgrading": [
+                    ["Release_Notes", "Release notes"],
+                    ["Upgrading_Project", "Upgrading Katello to 4.12"]
+                ],
+                "Quickstart": [
+                    ["Quickstart", "Quickstart guide"]
+                ],
+                "Deploying Foreman and Katello": [
+                    ["Planning_for_Project", "Planning for Foreman with Katello"],
+                    ["Installing_Server", "Installing Katello server"],
+                    ["Installing_Proxy", "Installing Smart Proxy with content"],
+                    ["Configuring_Load_Balancer", "Configuring Smart Proxies with a load balancer"],
+                    ["Deploying_Project_on_AWS", "Deploying Foreman on AWS"]
+                ],
+                "Administering Foreman server": [
+                    ["Administering_Project", "Administering Foreman"],
+                    ["Tuning_Performance", "Tuning performance"],
+                    ["Managing_Organizations_and_Locations", "Managing organizations and locations in Foreman"],
+                    ["Managing_Content", "Managing content"],
+                    ["Monitoring_Project", "Monitoring Foreman performance"]
+                ],
+                "Administering hosts": [
+                    ["Provisioning_Hosts", "Provisioning hosts"],
+                    ["Managing_Hosts", "Managing hosts"],
+                    ["Managing_Configurations_Ansible", "Configuring hosts by using Ansible"],
+                    ["Managing_Configurations_Puppet", "Configuring hosts by using Puppet"],
+                    ["Managing_Configurations_Salt", "Configuring hosts by using Salt"],
+                    ["Managing_Security_Compliance", "Managing security compliance"],
+                    ["Deploying_Hosts_AppCentric", "Application centric deployment"]
+                ]
+            }
+        },
+        {
+            "title": "Foreman on EL",
+            "header": "Foreman {FOREMAN_VER} on Enterprise Linux",
+            "filename": "index-foreman-el.html",
+            "sections": {
+                "Release notes and upgrading": [
+                    ["Release_Notes", "Release notes"],
+                    ["Upgrading_Project", "Upgrading Foreman to 3.10"]
```

Comment:
Not a big deal, but this could use `{FOREMAN_VER}` instead of 3.10.

Reply:
Addressed.